fe/fe-core/src/main/java/com/starrocks/load/Load.java

```diff
@@ -515,12 +515,30 @@ public static List<Column> getPartialUpateColumns(Table tbl, List<ImportColumnDe
 
     /**
      * @param excludedColumns: columns that the type should not be inferred from expr.
-     *                         Such as, the type of column from path is VARCHAR, whether it is in expr args or not,
+     *                         Such as, the type of column from path is VARCHAR,
+     *                         whether it is in expr args or not,
      *                         and column exists in both schema and expr args.
      */
-    private static void replaceSrcSlotDescType(Map<String, Expr> exprsByName, TupleDescriptor srcTupleDesc,
-                                               Set<String> excludedColumns) throws UserException {
+    private static void replaceSrcSlotDescType(Table tbl, Map<String, Expr> exprsByName, TupleDescriptor srcTupleDesc,
```

Comment:
You can run the daily test to check the cast

Reply:
pass the pr test